.jenkins/pytorch/print_sccache_log.py

```diff
@@ -6,7 +6,12 @@
     lines = f.readlines()
 
 for line in lines:
-    # Ignore errors from CPU instruction set or symbol existing testing
-    keywords = ['src.c', 'CheckSymbolExists.c']
+    # Ignore errors from CPU instruction set, symbol existing testing,
+    # or compilation error formatting
+    keywords = [
+        'src.c',
+        'CheckSymbolExists.c',
+        'test_compilation_error_formatting',
+    ]
     if all([keyword not in line for keyword in keywords]):
```

Comment:
```suggestion
    if all([keyword not in line for keyword in ignored_keywords]):
```

Reply:
why are you suggesting in your own PR? :-P

Reply:
> why are you suggesting in your own PR? :-P

@walterddr because your suggestion was insufficient to rename the variable, so I added a second suggestion to rename it in its usage so that I could use GitHub's "commit batch" feature to make the commit without having to go to the trouble of switching branches locally